code/modules/mob/living/carbon/xenomorph/abilities.dm

```diff
@@ -216,9 +216,15 @@
 
 /// Extra handling for adding the action for draggin functionality (for instant building)
 /datum/action/xeno_action/activable/secrete_resin/give_action(mob/living/L)
-	..()
+	. = ..()
 	if(!(CHECK_BITFIELD(SSticker.mode.flags_round_type, MODE_ALLOW_XENO_QUICKBUILD) && SSresinshaping.active))
 		return
+	var/mutable_appearance/build_maptext = mutable_appearance(layer = ACTION_LAYER_MAPTEXT)
+	build_maptext.pixel_x = 12
+	build_maptext.pixel_y = -5
+	build_maptext.maptext = MAPTEXT(SSresinshaping.get_building_points(owner))
+	visual_references[VREF_MUTABLE_BUILDING_COUNTER] = build_maptext
```

Comment:
???

Reply:
Moved to `give_action()` since it being in `New()` results in trying to access owner before owner is even assigned (in line 225).